app/controllers/concerns/success_message.rb

```diff
@@ -0,0 +1,15 @@
+module SuccessMessage
+  extend ActiveSupport::Concern
+
+  def details_flash_message(value)
```

Comment:
This is much better. I think we should make the method name a bit clearer so whoever uses it knows they're showing a **success** message. Maybe something like `publishable_flash_message` or something 🤷‍♂️